conf/messages.cy

```diff
@@ -108,6 +108,7 @@ site.submit = Cyflwyno
 site.submit.enquiry = Cyflwyno’ch ymholiad
 site.service_name = Ffurflen Gysylltu Asiantaeth y Swyddfa Brisio
 service.name = Ffurflen Gysylltu Asiantaeth y Swyddfa Brisio
+service.homePageUrl = /valuation-office-agency-contact-frontend/
```

Comment:
Wouldn't that belong more to the `application.conf` file ?

Reply:
It is used in default `play-frontend-govuk` template via `messages`

Reply:
okie dokes 👍